### DISCUSSION TITLE
How to install the Akeneo with CWP shell provided?

### DISCUSSION THREAD

**Discussion by weasam** [akeneo/pim-community-dev#16108] [Q&A]
Hello, forks, I stuck in the half way installing Akeneo in CWP-pro which Akeno is using composer not like other (prestashop, wordpress) just upload the  site files and create database. 

System: Centos 8 stream CWP delayed 4GB RAM, 4GB SWAP, 256GB SSD
Centos Control Webpanel(CWPpro version: 0.9.8.1116) with global Apache 2.4.52/ PHP version: 7.4.27

Akeneo is an excellent PIM solution for E-commerce website magento, prestashop. 

Akeneo requires composer to install the I follow the link to install the lasted version composer with root privileges user 
http://wiki.centos-webpanel.com/install-composer-php-dependency-manager

Akeneo installation link
https://docs.akeneo.com/latest/install_pim/manual/installation_ce.html

Here are the steps what I did:

1. install the php dependency Composer in root privileges follow the instruction
2. http://wiki.centos-webpanel.com/install-composer-php-dependency-manager
3. using a non-root privileges user:akatest from CWP and located to directory: public_html
4. Lest start the first command
`composer create-project akeneo/pim-community-standard /home/akatest/public_html/sub.mydomain.com "5.0.*@stable" `
---------------------------------------------COMMAND OUTPUT-----------------------------------------------
Creating a "akeneo/pim-community-standard" project at "./sub.mydomain.com"
Installing akeneo/pim-community-standard (v5.0.67)
  - Downloading akeneo/pim-community-standard (v5.0.67)
  - Installing akeneo/pim-community-standard (v5.0.67): Extracting archive
Created project in /home/aketest/sub.mydoamin.com
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - akeneo/pim-community-dev[v5.0.0, ..., v5.0.67] require ext-apcu * -> it is missing from your system. Install or enable PHP's apcu extension.
    - Root composer.json requires akeneo/pim-community-dev ^5.0.0 -> satisfiable by akeneo/pim-community-dev[v5.0.0, ..., v5.0.67].

To enable extensions, verify that they are enabled in your .ini files:
    - /usr/local/bin/php.ini
You can also run php --ini in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with --ignore-platform-req=ext-apcu to temporarily ignore these required extensions.

6.  Let's ignore the apcu extension, empty the files under diretory:sub.mydomain.com and start the new command

`composer create-project akeneo/pim-community-standard /home/aketest/sub.mydomain.com "5.0.*@stable" --ignore-platform-req=ext-apcu`

---------------------------------------------COMMAND OUTPUT-----------------------------------------------
Creating a "akeneo/pim-community-standard" project at "./pima.domain.com"
Installing akeneo/pim-community-standard (v5.0.67)
  - Installing akeneo/pim-community-standard (v5.0.67): Extracting archive
Created project in /home/aketest/sub.mydomain.com
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - akeneo/pim-community-dev[v5.0.0, ..., v5.0.67] require ext-imagick * -> it is missing from your system. Install or enable PHP's imagick extension.
    - Root composer.json requires akeneo/pim-community-dev ^5.0.0 -> satisfiable by akeneo/pim-community-dev[v5.0.0, ..., v5.0.67].

To enable extensions, verify that they are enabled in your .ini files:
    - /usr/local/bin/php.ini
You can also run php --ini in a terminal to see which files are used by PHP in CLI mode.
Alternatively, you can run Composer with --ignore-platform-req=ext-imagick to temporarily ignore these required extensions.

7. Next delete all files under directory:sub.mydomain.com and command also innore the extension:imagick
`composer create-project akeneo/pim-community-standard /home/domain/pima.domain.com "5.0.*@stable" --ignore-platform-req=ext-apcu --ignore-platform-req=ext-imagick`
---------------------------------------------COMMAND OUTPUT-----------------------------------------------
Creating a "akeneo/pim-community-standard" project at "./sub.mydomin.com"
https://repo.packagist.org could not be fully loaded (curl error 35 while downloading https://repo.packagist.org/packages.json: OpenSSL SSL_connect: SSL_ERROR_SYSCALL in connection to repo.packagist.org:443 ), package information was loaded from the local cache and may be out of date
Installing akeneo/pim-community-standard (v5.0.67)
  - Installing akeneo/pim-community-standard (v5.0.67): Extracting archive
Created project in /home/aketest/sub.mydomain.com
Loading composer repositories with package information
Updating dependencies
Lock file operations: 148 installs, 0 updates, 0 removals
  - Locking akeneo/pim-community-dev (v5.0.67)
  - Locking ass/xmlsecurity (v1.1.1)
  - Locking aws/aws-sdk-php (3.172.0)
  - Locking behat/transliterator (v1.3.0)
  - Locking box/spout (v2.7.3)
  - Locking composer/package-versions-deprecated (1.11.99.4)
  - Locking doctrine/annotations (v1.8.0)
  - Locking doctrine/cache (1.9.1)
  - Locking doctrine/collections (1.6.4)
  - Locking doctrine/common (v2.11.0)
  - Locking doctrine/data-fixtures (1.4.0)
  - Locking doctrine/dbal (v2.10.0)
  - Locking doctrine/doctrine-bundle (1.12.0)
  - Locking doctrine/doctrine-cache-bundle (1.4.0)
  - Locking doctrine/doctrine-fixtures-bundle (3.3.0)
  - Locking doctrine/doctrine-migrations-bundle (v1.3.2)
  - Locking doctrine/event-manager (1.1.0)
  - Locking doctrine/inflector (1.4.4)
  - Locking doctrine/instantiator (1.3.0)
  - Locking doctrine/lexer (1.2.1)
  - Locking doctrine/migrations (v1.8.1)
  - Locking doctrine/orm (v2.7.0)
  - Locking doctrine/persistence (1.2.0)
  - Locking doctrine/reflection (v1.0.0)
  - Locking dompdf/dompdf (v0.8.6)
  - Locking egulias/email-validator (3.1.2)
  - Locking elasticsearch/elasticsearch (v7.9.1)
  - Locking ezimuel/guzzlestreams (3.0.1)
  - Locking ezimuel/ringphp (1.2.0)
  - Locking firebase/php-jwt (v5.5.1)
  - Locking friendsofphp/proxy-manager-lts (v1.0.5)
  - Locking friendsofsymfony/jsrouting-bundle (2.1.1)
  - Locking friendsofsymfony/oauth-server-bundle (1.6.2)
  - Locking friendsofsymfony/oauth2-php (1.3.1)
  - Locking friendsofsymfony/rest-bundle (2.8.0)
  - Locking gedmo/doctrine-extensions (v2.4.37)
  - Locking google/auth (v1.18.0)
  - Locking google/cloud-core (v1.43.1)
  - Locking google/cloud-pubsub (v1.29.1)
  - Locking google/common-protos (1.4.0)
  - Locking google/gax (v1.10.0)
  - Locking google/grpc-gcp (v0.2.0)
  - Locking google/protobuf (v3.19.2)
  - Locking grpc/grpc (1.42.0)
  - Locking guzzlehttp/guzzle (6.5.5)
  - Locking guzzlehttp/promises (1.5.1)
  - Locking guzzlehttp/psr7 (1.8.3)
  - Locking imagine/imagine (v0.7.1)
  - Locking jdorn/sql-formatter (v1.2.17)
  - Locking justinrainbow/json-schema (5.2.11)
  - Locking laminas/laminas-code (4.5.1)
  - Locking league/flysystem (1.1.9)
  - Locking league/flysystem-ziparchive (1.0.3)
  - Locking league/mime-type-detection (1.9.0)
  - Locking liip/imagine-bundle (2.3.1)
  - Locking monolog/monolog (1.25.5)
  - Locking mtdowling/jmespath.php (2.6.1)
  - Locking oneup/flysystem-bundle (3.5.0)
  - Locking paragonie/random_compat (v2.0.20)
  - Locking phenx/php-font-lib (0.5.4)
  - Locking phenx/php-svg-lib (0.3.4)
  - Locking psr/cache (1.0.1)
  - Locking psr/container (1.1.2)
  - Locking psr/event-dispatcher (1.0.0)
  - Locking psr/http-message (1.0.1)
  - Locking psr/log (1.1.4)
  - Locking ralouphie/getallheaders (3.0.3)
  - Locking ramsey/uuid (3.9.6)
  - Locking react/promise (v2.8.0)
  - Locking rize/uri-template (0.3.4)
  - Locking sabberworm/php-css-parser (8.4.0)
  - Locking sensio/framework-extra-bundle (v5.5.6)
  - Locking swiftmailer/swiftmailer (v6.3.0)
  - Locking symfony/acl-bundle (v1.0.1)
  - Locking symfony/asset (v4.4.25)
  - Locking symfony/cache (v5.3.13)
  - Locking symfony/cache-contracts (v2.5.0)
  - Locking symfony/config (v4.4.25)
  - Locking symfony/console (v4.4.36)
  - Locking symfony/debug (v4.4.36)
  - Locking symfony/debug-bundle (v4.4.36)
  - Locking symfony/dependency-injection (v4.4.25)
  - Locking symfony/deprecation-contracts (v2.5.0)
  - Locking symfony/doctrine-bridge (v4.4.1)
  - Locking symfony/dotenv (v4.4.25)
  - Locking symfony/error-handler (v4.4.36)
  - Locking symfony/event-dispatcher (v4.4.25)
  - Locking symfony/event-dispatcher-contracts (v1.1.11)
  - Locking symfony/filesystem (v4.4.25)
  - Locking symfony/finder (v4.4.25)
  - Locking symfony/flex (v1.17.6)
  - Locking symfony/form (v4.4.25)
  - Locking symfony/framework-bundle (v4.4.1)
  - Locking symfony/http-client-contracts (v2.5.0)
  - Locking symfony/http-foundation (v4.4.25)
  - Locking symfony/http-kernel (v4.4.25)
  - Locking symfony/inflector (v5.4.0)
  - Locking symfony/intl (v4.4.25)
  - Locking symfony/lock (v4.4.25)
  - Locking symfony/messenger (v4.4.1)
  - Locking symfony/mime (v5.4.2)
  - Locking symfony/monolog-bridge (v5.2.12)
  - Locking symfony/monolog-bundle (v3.7.1)
  - Locking symfony/options-resolver (v4.4.25)
  - Locking symfony/orm-pack (v1.0.8)
  - Locking symfony/polyfill-apcu (v1.12.0)
  - Locking symfony/polyfill-ctype (v1.24.0)
  - Locking symfony/polyfill-iconv (v1.24.0)
  - Locking symfony/polyfill-intl-grapheme (v1.24.0)
  - Locking symfony/polyfill-intl-icu (v1.24.0)
  - Locking symfony/polyfill-intl-idn (v1.24.0)
  - Locking symfony/polyfill-intl-normalizer (v1.24.0)
  - Locking symfony/polyfill-mbstring (v1.24.0)
  - Locking symfony/polyfill-php72 (v1.24.0)
  - Locking symfony/polyfill-php73 (v1.24.0)
  - Locking symfony/polyfill-php80 (v1.24.0)
  - Locking symfony/polyfill-php81 (v1.24.0)
  - Locking symfony/process (v4.4.25)
  - Locking symfony/property-access (v4.4.25)
  - Locking symfony/proxy-manager-bridge (v4.4.25)
  - Locking symfony/requirements-checker (v1.1.8)
  - Locking symfony/routing (v4.4.25)
  - Locking symfony/security-acl (v3.0.2)
  - Locking symfony/security-bundle (v4.4.25)
  - Locking symfony/security-core (v4.4.25)
  - Locking symfony/security-csrf (v4.4.25)
  - Locking symfony/security-guard (v4.4.27)
  - Locking symfony/security-http (v4.4.25)
  - Locking symfony/serializer (v4.4.36)
  - Locking symfony/service-contracts (v2.5.0)
  - Locking symfony/string (v5.4.2)
  - Locking symfony/swiftmailer-bundle (v3.5.3)
  - Locking symfony/templating (v4.4.25)
  - Locking symfony/translation (v4.4.25)
  - Locking symfony/translation-contracts (v2.5.0)
  - Locking symfony/twig-bridge (v4.4.36)
  - Locking symfony/twig-bundle (v4.4.25)
  - Locking symfony/validator (v4.4.25)
  - Locking symfony/var-dumper (v5.4.2)
  - Locking symfony/var-exporter (v5.4.2)
  - Locking symfony/web-profiler-bundle (v4.4.31)
  - Locking symfony/web-server-bundle (v4.4.27)
  - Locking symfony/yaml (v4.4.25)
  - Locking twig/extensions (v1.3.0)
  - Locking twig/twig (v1.44.1)
  - Locking webmozart/assert (1.5.0)
  - Locking willdurand/jsonp-callback-validator (v1.1.0)
  - Locking willdurand/negotiation (v2.3.1)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 148 installs, 0 updates, 0 removals
  - Downloading webmozart/assert (1.5.0)
  - Downloading twig/twig (v1.44.1)
  - Downloading twig/extensions (v1.3.0)
  - Downloading symfony/security-acl (v3.0.2)
  - Downloading symfony/requirements-checker (v1.1.8)
  - Downloading symfony/polyfill-apcu (v1.12.0)
  - Downloading doctrine/annotations (v1.8.0)
  - Downloading doctrine/reflection (v1.0.0)
  - Downloading doctrine/event-manager (1.1.0)
  - Downloading doctrine/collections (1.6.4)
  - Downloading doctrine/cache (1.9.1)
  - Downloading doctrine/persistence (1.2.0)
  - Downloading doctrine/instantiator (1.3.0)
  - Downloading doctrine/dbal (v2.10.0)
  - Downloading doctrine/common (v2.11.0)
  - Downloading doctrine/orm (v2.7.0)
  - Downloading symfony/framework-bundle (v4.4.1)
  - Downloading doctrine/migrations (v1.8.1)
  - Downloading symfony/doctrine-bridge (v4.4.1)
  - Downloading jdorn/sql-formatter (v1.2.17)
  - Downloading doctrine/doctrine-cache-bundle (1.4.0)
  - Downloading doctrine/doctrine-bundle (1.12.0)
  - Downloading doctrine/doctrine-migrations-bundle (v1.3.2)
  - Downloading symfony/orm-pack (v1.0.8)
  - Downloading monolog/monolog (1.25.5)
  - Downloading symfony/messenger (v4.4.1)
  - Downloading symfony/acl-bundle (v1.0.1)
  - Downloading sensio/framework-extra-bundle (v5.5.6)
  - Downloading oneup/flysystem-bundle (3.5.0)
  - Downloading imagine/imagine (v0.7.1)
  - Downloading liip/imagine-bundle (2.3.1)
  - Downloading league/flysystem-ziparchive (1.0.3)
  - Downloading google/cloud-pubsub (v1.29.1)
  - Downloading behat/transliterator (v1.3.0)
  - Downloading gedmo/doctrine-extensions (v2.4.37)
  - Downloading willdurand/negotiation (v2.3.1)
  - Downloading willdurand/jsonp-callback-validator (v1.1.0)
  - Downloading friendsofsymfony/rest-bundle (2.8.0)
  - Downloading friendsofsymfony/oauth-server-bundle (1.6.2)
  - Downloading friendsofsymfony/jsrouting-bundle (2.1.1)
  - Downloading elasticsearch/elasticsearch (v7.9.1)
  - Downloading dompdf/dompdf (v0.8.6)
  - Downloading doctrine/data-fixtures (1.4.0)
  - Downloading doctrine/doctrine-fixtures-bundle (3.3.0)
  - Downloading box/spout (v2.7.3)
  - Downloading aws/aws-sdk-php (3.172.0)
  - Downloading ass/xmlsecurity (v1.1.1)
  0/47 [>---------------------------]   0%  - Downloading symfony/requirements-checker (v1.1.8)
  - Downloading doctrine/reflection (v1.0.0)
  - Downloading doctrine/collections (1.6.4)
  - Downloading doctrine/cache (1.9.1)
  - Downloading webmozart/assert (1.5.0)
  - Downloading doctrine/persistence (1.2.0)
  - Downloading symfony/security-acl (v3.0.2)
  - Downloading doctrine/event-manager (1.1.0)
  - Downloading twig/extensions (v1.3.0)
  - Downloading twig/twig (v1.44.1)
  - Downloading symfony/polyfill-apcu (v1.12.0)
  - Downloading doctrine/annotations (v1.8.0)
  - Downloading symfony/requirements-checker (v1.1.8)
  - Downloading doctrine/reflection (v1.0.0)
  - Downloading doctrine/collections (1.6.4)
  - Downloading doctrine/cache (1.9.1)
  - Installing composer/package-versions-deprecated (1.11.99.4): Extracting archive
  - Installing symfony/flex (v1.17.6): Extracting archive
  - Installing symfony/polyfill-ctype (v1.24.0): Extracting archive
  - Installing webmozart/assert (1.5.0): Extracting archive
  - Installing twig/twig (v1.44.1): Extracting archive
  - Installing twig/extensions (v1.3.0): Extracting archive
  - Installing symfony/yaml (v4.4.25): Extracting archive
  - Installing symfony/translation-contracts (v2.5.0): Extracting archive
  - Installing symfony/polyfill-mbstring (v1.24.0): Extracting archive
  - Installing symfony/validator (v4.4.25): Extracting archive
  - Installing symfony/polyfill-php80 (v1.24.0): Extracting archive
  - Installing symfony/twig-bridge (v4.4.36): Extracting archive
  - Installing symfony/polyfill-php73 (v1.24.0): Extracting archive
  - Installing symfony/polyfill-php72 (v1.24.0): Extracting archive
  - Installing symfony/polyfill-intl-normalizer (v1.24.0): Extracting archive
  - Installing symfony/polyfill-intl-idn (v1.24.0): Extracting archive
  - Installing symfony/deprecation-contracts (v2.5.0): Extracting archive
  - Installing symfony/mime (v5.4.2): Extracting archive
  - Installing symfony/http-foundation (v4.4.25): Extracting archive
  - Installing symfony/http-client-contracts (v2.5.0): Extracting archive
  - Installing symfony/event-dispatcher-contracts (v1.1.11): Extracting archive
  - Installing symfony/event-dispatcher (v4.4.25): Extracting archive
  - Installing symfony/var-dumper (v5.4.2): Extracting archive
  - Installing psr/log (1.1.4): Extracting archive
  - Installing symfony/debug (v4.4.36): Extracting archive
  - Installing symfony/error-handler (v4.4.36): Extracting archive
  - Installing symfony/http-kernel (v4.4.25): Extracting archive
  - Installing symfony/twig-bundle (v4.4.25): Extracting archive
  - Installing symfony/translation (v4.4.25): Extracting archive
  - Installing symfony/templating (v4.4.25): Extracting archive
  - Installing psr/container (1.1.2): Extracting archive
  - Installing symfony/service-contracts (v2.5.0): Extracting archive
  - Installing symfony/dependency-injection (v4.4.25): Extracting archive
  - Installing symfony/polyfill-php81 (v1.24.0): Extracting archive
  - Installing symfony/filesystem (v4.4.25): Extracting archive
  - Installing symfony/config (v4.4.25): Extracting archive
  - Installing symfony/polyfill-iconv (v1.24.0): Extracting archive
  - Installing doctrine/lexer (1.2.1): Extracting archive
  - Installing egulias/email-validator (3.1.2): Extracting archive
  - Installing swiftmailer/swiftmailer (v6.3.0): Extracting archive
  - Installing symfony/swiftmailer-bundle (v3.5.3): Extracting archive
  - Installing symfony/security-core (v4.4.25): Extracting archive
  - Installing symfony/polyfill-intl-grapheme (v1.24.0): Extracting archive
  - Installing symfony/string (v5.4.2): Extracting archive
  - Installing symfony/inflector (v5.4.0): Extracting archive
  - Installing symfony/property-access (v4.4.25): Extracting archive
  - Installing symfony/security-http (v4.4.25): Extracting archive
  - Installing symfony/security-csrf (v4.4.25): Extracting archive
  - Installing symfony/security-guard (v4.4.27): Extracting archive
  - Installing symfony/security-bundle (v4.4.25): Extracting archive
  - Installing symfony/security-acl (v3.0.2): Extracting archive
  - Installing symfony/routing (v4.4.25): Extracting archive
  - Installing symfony/requirements-checker (v1.1.8): Extracting archive
  - Installing laminas/laminas-code (4.5.1): Extracting archive
  - Installing friendsofphp/proxy-manager-lts (v1.0.5): Extracting archive
  - Installing symfony/proxy-manager-bridge (v4.4.25): Extracting archive
  - Installing symfony/process (v4.4.25): Extracting archive
  - Installing symfony/polyfill-apcu (v1.12.0): Extracting archive
  - Installing symfony/console (v4.4.36): Extracting archive
  - Installing doctrine/annotations (v1.8.0): Extracting archive
  - Installing doctrine/reflection (v1.0.0): Extracting archive
  - Installing doctrine/event-manager (1.1.0): Extracting archive
  - Installing doctrine/collections (1.6.4): Extracting archive
  - Installing doctrine/cache (1.9.1): Extracting archive
  - Installing doctrine/persistence (1.2.0): Extracting archive
  - Installing doctrine/instantiator (1.3.0): Extracting archive
  - Installing doctrine/dbal (v2.10.0): Extracting archive
  - Installing doctrine/inflector (1.4.4): Extracting archive
  - Installing doctrine/common (v2.11.0): Extracting archive
  - Installing doctrine/orm (v2.7.0): Extracting archive
  - Installing symfony/finder (v4.4.25): Extracting archive
  - Installing symfony/var-exporter (v5.4.2): Extracting archive
  - Installing psr/cache (1.0.1): Extracting archive
  - Installing symfony/cache-contracts (v2.5.0): Extracting archive
  - Installing symfony/cache (v5.3.13): Extracting archive
  - Installing symfony/framework-bundle (v4.4.1): Extracting archive
  - Installing doctrine/migrations (v1.8.1): Extracting archive
  - Installing symfony/doctrine-bridge (v4.4.1): Extracting archive
  - Installing jdorn/sql-formatter (v1.2.17): Extracting archive
  - Installing doctrine/doctrine-cache-bundle (1.4.0): Extracting archive
  - Installing doctrine/doctrine-bundle (1.12.0): Extracting archive
  - Installing doctrine/doctrine-migrations-bundle (v1.3.2): Extracting archive
  - Installing symfony/orm-pack (v1.0.8): Extracting archive
  - Installing symfony/options-resolver (v4.4.25): Extracting archive
  - Installing monolog/monolog (1.25.5): Extracting archive
  - Installing symfony/monolog-bridge (v5.2.12): Extracting archive
  - Installing symfony/monolog-bundle (v3.7.1): Extracting archive
  - Installing symfony/messenger (v4.4.1): Extracting archive
  - Installing symfony/lock (v4.4.25): Extracting archive
  - Installing symfony/polyfill-intl-icu (v1.24.0): Extracting archive
  - Installing symfony/intl (v4.4.25): Extracting archive
  - Installing symfony/form (v4.4.25): Extracting archive
  - Installing symfony/dotenv (v4.4.25): Extracting archive
  - Installing symfony/asset (v4.4.25): Extracting archive
  - Installing symfony/acl-bundle (v1.0.1): Extracting archive
  - Installing sensio/framework-extra-bundle (v5.5.6): Extracting archive
  - Installing paragonie/random_compat (v2.0.20): Extracting archive
  - Installing ramsey/uuid (3.9.6): Extracting archive
  - Installing psr/event-dispatcher (1.0.0): Extracting archive
  - Installing league/mime-type-detection (1.9.0): Extracting archive
  - Installing league/flysystem (1.1.9): Extracting archive
  - Installing oneup/flysystem-bundle (3.5.0): Extracting archive
  - Installing imagine/imagine (v0.7.1): Extracting archive
  - Installing liip/imagine-bundle (2.3.1): Extracting archive
  - Installing league/flysystem-ziparchive (1.0.3): Extracting archive
  - Installing justinrainbow/json-schema (5.2.11): Extracting archive
  - Installing ralouphie/getallheaders (3.0.3): Extracting archive
  - Installing psr/http-message (1.0.1): Extracting archive
  - Installing guzzlehttp/psr7 (1.8.3): Extracting archive
  - Installing guzzlehttp/promises (1.5.1): Extracting archive
  - Installing guzzlehttp/guzzle (6.5.5): Extracting archive
  - Installing grpc/grpc (1.42.0): Extracting archive
  - Installing google/protobuf (v3.19.2): Extracting archive
  - Installing firebase/php-jwt (v5.5.1): Extracting archive
  - Installing google/auth (v1.18.0): Extracting archive
  - Installing google/grpc-gcp (v0.2.0): Extracting archive
  - Installing google/common-protos (1.4.0): Extracting archive
  - Installing google/gax (v1.10.0): Extracting archive
  - Installing rize/uri-template (0.3.4): Extracting archive
  - Installing google/cloud-core (v1.43.1): Extracting archive
  - Installing google/cloud-pubsub (v1.29.1): Extracting archive
  - Installing behat/transliterator (v1.3.0): Extracting archive
  - Installing gedmo/doctrine-extensions (v2.4.37): Extracting archive
  - Installing willdurand/negotiation (v2.3.1): Extracting archive
  - Installing willdurand/jsonp-callback-validator (v1.1.0): Extracting archive
  - Installing friendsofsymfony/rest-bundle (2.8.0): Extracting archive
  - Installing friendsofsymfony/oauth2-php (1.3.1): Extracting archive
  - Installing friendsofsymfony/oauth-server-bundle (1.6.2): Extracting archive
  - Installing symfony/serializer (v4.4.36): Extracting archive
  - Installing friendsofsymfony/jsrouting-bundle (2.1.1): Extracting archive
  - Installing react/promise (v2.8.0): Extracting archive
  - Installing ezimuel/guzzlestreams (3.0.1): Extracting archive
  - Installing ezimuel/ringphp (1.2.0): Extracting archive
  - Installing elasticsearch/elasticsearch (v7.9.1): Extracting archive
  - Installing sabberworm/php-css-parser (8.4.0): Extracting archive
  - Installing phenx/php-svg-lib (0.3.4): Extracting archive
  - Installing phenx/php-font-lib (0.5.4): Extracting archive
  - Installing dompdf/dompdf (v0.8.6): Extracting archive
  - Installing doctrine/data-fixtures (1.4.0): Extracting archive
  - Installing doctrine/doctrine-fixtures-bundle (3.3.0): Extracting archive
  - Installing box/spout (v2.7.3): Extracting archive
  - Installing mtdowling/jmespath.php (2.6.1): Extracting archive
  - Installing aws/aws-sdk-php (3.172.0): Extracting archive
  - Installing ass/xmlsecurity (v1.1.1): Extracting archive
  - Installing akeneo/pim-community-dev (v5.0.67): Extracting archive
  - Installing symfony/debug-bundle (v4.4.36): Extracting archive
  - Installing symfony/web-profiler-bundle (v4.4.31): Extracting archive
  - Installing symfony/web-server-bundle (v4.4.27): Extracting archive
Package doctrine/doctrine-cache-bundle is abandoned, you should avoid using it. No replacement was suggested.
Package doctrine/reflection is abandoned, you should avoid using it. Use roave/better-reflection instead.
Package swiftmailer/swiftmailer is abandoned, you should avoid using it. Use symfony/mailer instead.
Package symfony/inflector is abandoned, you should avoid using it. Use use `EnglishInflector` from the String component instead instead.
Package twig/extensions is abandoned, you should avoid using it. No replacement was suggested.
Generating autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
78 packages you are using are looking for funding.
Use the `composer fund` command to find out more!
Symfony recipes are disabled: "symfony/flex" not found in the root composer.json

> bash vendor/akeneo/pim-community-dev/std-build/install-required-files.sh
Project configuration is disabled: "symfony/flex" not found in the root composer.json
> bash vendor/akeneo/pim-community-dev/std-build/install-required-files.sh
src/ directory already exists. Not preparing the directory content.

8. untill now I don't want to do, I start look over the answer from google, no any could resolve this

What I'm thinking is composer is using the system PHP will use the PHP compiled in PHP selectors or the configuration in Vhost

Anyone have any ideas?  **please please help Appreciated**

**Comment by sebadamus** [comment]
Hi, could you get it to work? I am also stuck but on ubuntu with Akeneo Community Edition :-(

**Reply by weasam** [reply]
I tried Cyberpanel with Shell still could not install even the apcu& imagick dependencies installed, then the error not come up with dependencies error
here is my code
`/usr/local/lsws/lsphp74/bin/php /usr/bin/composer create-project akeneo/pim-community-standard /home/pima.domain.com/public_html "5.0.*@stable" `

The Output as below
Symfony recipes are disabled: "symfony/flex" not found in the root composer.json

		> bash vendor/akeneo/pim-community-dev/std-build/install-required-files.sh
		Project configuration is disabled: "symfony/flex" not found in the root composer.json
		> bash vendor/akeneo/pim-community-dev/std-build/install-required-files.sh
		src/ directory already exists. Not preparing the directory content.

Obviously is the composer.json is configure incorrectly by Akeneo PIM, need to fix" src/ directory already exists
I'm trying another attempt is using Bitnami Virtual machine package then migrate to remote server

**Reply by sebadamus** [reply]
I (modestly) think those are kind of warnings and not errors (I mean next to "The Output as below...")

After you run the 'composer create-project...' command, did you run this?: `NO_DOCKER=true make prod`

**Reply by weasam** [reply]
`nano .env`  ##setting database
`bin/console pim:installer:check-requirements`   ##check requiments, everything is ok except small warning
`NO_DOCKER=true make prod`
Output:  
1. Database does not exist yet
2. SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo failed: Name or service not known
##Docker MYSQL 8.0 , other php app could connect, firewall disabled

I guess It is the NodeJS 16 or composer 2.2 issue
